content/eng/manifesto/aap_2014.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Aam Aadmi Party-2014"
+title: "AAP 2014 Manifesto"
+date: 2019-04-04
+type: post
+categories:
+
+- manifesto
+  tags:
+- AAP
+- 2014
+  draft: true
+
+# url: /about/
+
+# thumbnail: https://raw.githubusercontent.com/jesselau76/hugo-w3-simple/master/exampleSite/images/thumb-rigo-erives-594315-unsplash.jpg
+---
+
+**Manifesto** - What does manifesto represent. _Manifesto is a public statement of someone's policies, views and intention do something which political parties put in front of people before elections._
+
+Why do **Political Parities Need Manifesto** - _Manifesto functions as both a statement of a principles and a bold, sometimes
+rebellious call to action. By causing people to evaluate the gap between those principles and their current reality, the manifesto challenges assumptions, fosters commitment, and provokes change._
+
+Here, I will list down notable points **Aam Aadmi Party** had put in front of people before 2014 Lok Sabha Elections:
```

Comment:
put general election than lok sabha (or explain it) outside india won't understand it